site/content/en/docs/tasks/manage/monitor_pending_workloads/pending_workloads_on_demand.md

```diff
@@ -26,17 +26,9 @@ Make sure the following conditions are met:
 
 VisibilityOnDemand is an `Alpha` feature disabled by default. To use the visibility API  change [the feature gates configuration](/docs/installation/#change-the-feature-gates-configuration) and set `VisibilityOnDemand=true`.
 
-### Install the visibility API
-
-To install the visibility API, run the following command
-
-```shell
-kubectl apply --server-side -f https://github.com/kubernetes-sigs/kueue/releases/download/{{< param "version" >}}/visibility-api.yaml
-```
-
 ## Monitor pending workloads on demand
 
-{{< feature-state state="stable" for_version="v0.6" >}}
+{{< feature-state state="alpha" for_version="v0.6" >}}
```

Comment:
Please move the fix to dedicated PR, we will want to cherry-pick this.

Reply:
Done #3088.